### PR TITLE
test: Add test to check hostname length (HMS-5185)

### PR DESCRIPTION
**Pull request by regexowl**
This adds a test to check the length of the hostname.<br/><br/>JIRA: [HMS-5185](https://issues.redhat.com/browse/HMS-5185)

### CODE REVIEW

**Comment by regexowl** [comment]
Also updated the validator a tiny bit to reflect the "up to 64 lower-case characters" part.

File: src/test/Components/CreateImageWizard/steps/Hostname/Hostname.test.tsx [hunk 1]
```diff
@@ -103,6 +103,21 @@ describe('Step Hostname', () => {
...
     expect(screen.queryByText(/Invalid hostname/)).not.toBeInTheDocument();
   });
+
+  test('hostname is invalid for more than 64 chars', async () => {
+    await renderCreateMode();
+    await goToHostnameStep();
+    const nextButton = await getNextButton();
+
+    // enter invalid hostname
+    const invalidHostname = 'a'.repeat(65);
```
Comment (mgold1234) [inline src/test/Components/CreateImageWizard/steps/Hostname/Hostname.test.tsx:113]:
```suggestion
    const invalidHostName = 'a'.repeat(65);
```

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**